expression/builtin_vectorized.go

```diff
@@ -138,18 +147,25 @@ func (c *vecRowConverter) evalReal(row chunk.Row) (val float64, isNull bool, err
 	if !c.builtinFunc.vectorized() {
 		return c.builtinFunc.evalReal(row)
 	}
-	var sel [1]int
-	sel[0] = row.Idx()
 	var buf *chunk.Column
 	if buf, err = c.builtinFunc.get(types.ETInt, 1); err != nil {
```

Comment:
It's still`types.ETInt` here, is that correct?

Reply:
Now the converter has been removed.